src/engines/L3/element.js

```diff
@@ -266,65 +258,63 @@ const Element = {
     }
 
     // correct for default white nodes (but not for text nodes)
-    if (this.props.props['color'] === undefined && !('__textnode' in props)) {
+    if (this.props.props['color'] === undefined && '__textnode' in props === false) {
       this.props.props['color'] = 0
     }
 
     this.node = props.__textnode
       ? renderer.createTextNode({ ...textDefaults, ...this.props.props })
       : renderer.createNode(this.props.props)
 
-    if (props['@loaded']) {
+    if ('@loaded' in props === true && typeof props['@loaded'] === 'function') {
```

Comment:
👀 

is `'@loaded' in props === true` faster then an undefined check?

Reply:
No it's not :smile: .. changing to `undefined` check